README.md

```diff
@@ -421,9 +416,8 @@ abstract class PersonDao {
 
 ### Queries
 Method signatures turn into query methods by adding the `@Query()` annotation with the query in parenthesis to them.
-Be patient about the correctness of your SQL statements.
-They are only partly validated while generating the code.
-These queries have to return either a `Future` or a `Stream` of an entity or `void`.
+Your SQL queries will be validated completely while generating the code.
+These queries have to return either a `Future` or a `Stream` of an entity, a view, a primitive value or `void`.
```

Comment:
This will need another PR but it should be as simple as adding another Queryable subtype and using the same processing as for entities/views. The type checking and the mapping already exists.